src/ripple/app/tx/impl/Transactor.cpp

```diff
@@ -350,6 +350,23 @@ Transactor::calculateBaseFee(ReadView const& view, STTx const& tx)
 
     XRPAmount accumulator = baseFee;
 
+    if (view.rules().enabled(featureHooks) &&
+        tx.isFieldPresent(sfHookParameters))
+    {
+        uint64_t paramBytes = 0;
+        auto const& params = tx.getFieldArray(sfHookParameters);
+        for (auto const& param : params)
+        {
+            paramBytes += (param.isFieldPresent(sfHookParameterName)
+                               ? param.getFieldVL(sfHookParameterName).size()
+                               : 0) +
+                (param.isFieldPresent(sfHookParameterValue)
+                     ? param.getFieldVL(sfHookParameterValue).size()
+                     : 0);
+        }
+        accumulator += XRPAmount{static_cast<XRPAmount>(paramBytes)};
```

Comment:
these used to have overflow checks didn't they? what happened to that logic?

Reply:
The overflow checks were in SetHook.cpp but not in Invoke.cpp. I can add them here though